app/views/member_mailer/added_project.html.erb

```diff
@@ -0,0 +1,10 @@
+<%=raw I18n.t(:'mail_member_added_project.body.added_by', project: @project.name, user: User.current.name) %>
```

Comment:
Why do you need `raw` here? I only see a plain string in `added_by` so it should be escaped appropriately

Reply:
No clue, actually. Probably C&P stupidity. Thanks for catching.